src/plugins/workspace/server/saved_objects/saved_objects_wrapper_for_check_workspace_conflict.ts

```diff
@@ -211,6 +262,18 @@ export class WorkspaceConflictSavedObjectsClientWrapper {
         ...realBulkCreateResult,
         saved_objects: [
           ...objectsConflictWithWorkspace,
+          ...disallowedSavedObjects.map((item) => ({
+            references: [],
+            id: '',
```

Comment:
Just curious why need to set default value for `id` and `references`
```suggestion
            references: [],
            id: '',
```